.coderabbit-current.yaml

```
@@ -0,0 +1,457 @@
+ # Default: "en-US"
+ language: en-US
+ # Set the tone of reviews and chat.
+ # Default: ""
+ tone_instructions: '' # not in base
+ # Enable early-access features.
+ # Default: false
+ early_access: false # not in base
+ # Enable free tier features for users not on a paid plan.
+ # Default: true
+ enable_free_tier: true # not in base
+ tone_instructions: ''
```

YAMLlint (1.37.1)

[error] duplication of key "tone_instructions" in mapping

(key-duplicates)